jetson/ra_kinematics/mrover_arm.cpp

```diff
@@ -697,6 +697,11 @@ void StandardArm::lock_joints_callback(std::string channel, LockJoints msg) {
 
 void StandardArm::arm_preset_callback(std::string channel, ArmPreset msg) {
     std::vector<double> angles = arm_state.get_preset_position(msg.preset);
+
+    // invalid preset
+    if (angles.size() == 0) {
```

Comment:
Did you test this? I think get_preset_position might throw an exception if we give it something invalid...

Reply:
I did not test it and turns out my changes don't even compile lol

But this part is fine, the getter returns an empty vector if we give it an invalid preset.

Reply:
Oh that's kinda wack, the online documentation for at() says it would throw an error (though that's not how I previously remembered it working).

Reply:
Nvm I'm dumb, you had to change it to avoid throwing that exception. Looks good to me then

Reply:
It does throw an error, but I got rid of .at()

Reply:
yep yep